com.microsoft.mrtk.audio/Effects/AudioInfluencerController.cs

```diff
@@ -105,12 +110,6 @@ public float MaxDistance
         /// </summary>
         private DateTime lastUpdate = DateTime.MinValue;
 
-        /// <summary>
-        /// The source of the audio.
-        /// </summary>
-        [SerializeField]
-        private AudioSource audioSource;
```

Comment:
Is there any scenario where I'd want to reference an `AudioSource` on a different GameObject?

Reply:
i do not believe so. this was a holdover from mrtk2 where i thought it might be useful.